Add news announcement on new docs site (#82)
nunit/nunit.github.io: _posts/2020-07-26-new-docs-site-launched.md

---
layout: post
title:  "NUnit Launches docs.nunit.org, Welcomes Sean Killeen as Docs Project Lead"

> date:   2020-02-15 12:00:00 -0000

Author: ```suggestion
date:   2020-97-26 12:00:00 -0000
```

Wrong date.

categories: news update nunit docs
---

> NUnit's documentation has moved from its former GitHub wiki to a new site at https://docs.nunit.org. The site is now built from the content at [nunit/docs GitHub repository](https://github.com/nunit/docs).

Author: ```suggestion
NUnit's documentation has moved from its former GitHub wiki to a new site at https://docs.nunit.org. The site is now built from the content at the [nunit/docs GitHub repository](https://github.com/nunit/docs).
```

> [Sean Killeen](https://SeanKilleen.com) has joined the NUnit team as the new lead of the docs project. We welcome Sean to the project, and look forward to seeing everyone's great contributions to the docs going forward!

Reviewer: Ultra nitpicking here. A conjunction like "and" normally doesn't have a comma if it is followed by a dependent clause. I'd remove the comma or change to "and we look forward" if it was me.

Reviewer: (I also try to avoid using the same word twice too close together like "forward" because it can be distracting. This one might just be a "me" thing though.)

Author: @jnm2 I agree. I wrote it up quick. I'll make a small edit.

> [Sean Killeen](https://SeanKilleen.com) has joined the NUnit team as the new lead of the docs project. We welcome Sean to the project, and look forward to seeing everyone's great contributions to the docs going forward!

Author: ```suggestion
[Sean Killeen](https://SeanKilleen.com) has joined the NUnit team as the new lead of the docs project. We welcome Sean to the project, and are excited to see everyone's great contributions to the docs going forward!
```
